src/ZkFold/Symbolic/Data/ByteString.hs

```diff
@@ -232,24 +211,16 @@ instance (Symbolic c, KnownNat n) => BoolType (ByteString n c) where
 -- 4. @wordSize@ divides @n@.
 --
 
-instance
-  ( Symbolic c
-  , KnownNat wordSize
-  , (Div n wordSize) * wordSize ~ n
-  ) => ToWords (ByteString n c) (ByteString wordSize c) where
-    toWords (ByteString bits) = Haskell.map (ByteString . packed) $ V.fromVector . V.chunks @(Div n wordSize) @wordSize $ unpacked bits
+toWords :: forall n wordSize c. (Symbolic c, KnownNat wordSize, Div n wordSize * wordSize ~ n) => ByteString n c -> Vector (Div n wordSize) (ByteString wordSize c)
```

Comment:
It's better to avoid `Div`, like this:
```haskell
toWords :: (Symbolic c, KnownNat wordSize) => ByteString (m * wordSize) c -> Vector m (ByteString wordSize c)
```